docker-compose.yml

```diff
@@ -30,6 +30,17 @@ services:
     restart: on-failure
     depends_on:
       - postgres
+  proxy-carthagenet:
+    build:
+      context: .
+      dockerfile: build/package/proxy-carthagenet.Dockerfile
```

Comment:
Why not targeting the proxy.Dockerfile and adding environment variable for network in the `environment` part 
 of the docker-compose ?

Reply:
agreed